### PR TITLE
Adding Babylonian method in go

### PR DESCRIPTION
**Pull request by jdalrymple**

### CODE REVIEW

File: code/mathematical-algorithms/babylonian_method/babylonian_method.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package main
```
Comment (yatharthshahjpr) [inline code/mathematical-algorithms/babylonian_method/babylonian_method.go]: Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

**Comment by jdalrymple** [comment]
Will do!

File: code/mathematical-algorithms/babylonian_method/babylonian_method.go (hunk 1 above)
Comment (AdiChat) [reply code/mathematical-algorithms/babylonian_method/babylonian_method.go]: The comment is not a must. Note it 👍

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**